packages/manager/modules/nutanix/src/dashboard/nodes/template.html

```diff
@@ -0,0 +1,54 @@
+<oui-datagrid rows="$ctrl.nodes" data-page-size="25" class="mt-5">
+    <oui-datagrid-topbar>
+        <oui-spinner data-ng-if="ctrl.delete.loading"></oui-spinner>
+    </oui-datagrid-topbar>
+    <oui-datagrid-column
+        title="'nutanix_dashboard_nodes_list_display_name' | translate"
+        property="displayName"
+        data-sortable
+        data-searchable
+        data-filterable
+    >
+        <a
+            class="oui-link"
+            data-ui-sref="{{:: $ctrl.constructor.getNodeDetailsState($row.name) }}"
+            data-ng-bind="$row.displayName"
+        >
+        </a>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        title="'nutanix_dashboard_nodes_list_type_of_server' | translate"
+        property="commercialRange"
+        data-sortable
+        data-searchable
+        data-filterable
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        title="'nutanix_dashboard_nodes_list_status' | translate"
+        property="state"
+        data-sortable
+        data-searchable
+        data-filterable
+    >
+        <span
+            class="oui-badge"
+            data-ng-class="{
+                'oui-badge_success': $row.state === 'OK',
+                'oui-badge_warning': $row.state === 'ERROR',
+                'oui-badge_error': $row.state === 'HACKED' || $row.state === 'HACKED_BLOCKED'
```

Comment:
To be consistent with others implementation, could be great to have this :
```suggestion
                'oui-badge_success': $row.isSuccess(),
                'oui-badge_warning': $row.isWarning(),
                'oui-badge_error': $row.isError()
```

Reply:
I agree with your suggestions. But I can not so it. Dedicated servers are used in Nutanix cluster as nodes. Same old code of dedicated server is being reused in Nutanix node. Making this change here will impact dedicated server too. I would like to avoid this now and take it as a separate refactoring task.